msi/publish.ps1

```diff
@@ -0,0 +1,6 @@
+# #!/bin/bash -ex
```

Comment:
I guess this file in the current state is more a placeholder?
Publishing part will probably mean uploading generated files on azure with azure-cli

Reply:
on azure storage container defined [here](https://github.com/jenkins-infra/azure/blob/master/plans/releases-storage.tf#L164-L142)

Reply:
Yes, that's correct, this is very much a placeholder.

Reply:
How do I use those azure storage container definitions? Are they exported as environment variables?